utils/http.go

```diff
@@ -66,13 +67,28 @@ func (root *rootHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 	if err := root.handler(ctx, w, r); err != nil {
-		logrus.Error("[Notary Server] ", err.Error())
+		if err, ok := err.(errcode.Error); ok {
+			logrus.Errorf(
+				"[Notary Server] %d %s %s",
+				err.Code.Descriptor().HTTPStatusCode,
+				r.Method,
+				r.URL.Path,
+			)
+		} else {
+			logrus.Errorf(
+				"[Notary Server] 5XX %s %s %s",
```

Comment:
In this instance, it will likely always be a 500, but ServeJSON actually makes that decision. I put 5XX because if we ever see this, it's something that needs to be fixed ASAP, as it's essentially an unexpected, unhandled error.